static/bidder-info/viewdeos.yaml

```diff
@@ -3,8 +3,3 @@ endpoint: "http://ghb.sync.viewdeos.com/pbs/ortb"
 maintainer:
   email: "contact@viewdeos.com"
 gvlVendorID: 924
-userSync:
-  # viewdeos supports user syncing, but requires configuration by the host. contact this
-  # bidder directly at the email address in this file to ask about enabling user sync.
-  supports:
-    - redirect
```

Comment:
I'd live for Viewdeos or Adtelligent to comment before we make this change. Is there a case to enable for an alias only and not the parent? Viewdeos declares it can support redirect user sync whereas Adtelligent declares it can support iframe.